### PR TITLE
Fix tests

### PR DESCRIPTION
**Pull request by lucasmoura**
## Why is this needed?
Fix some integration tests for the release

## Test Steps
Check that the affected integration tests are passing

---

- [x] *(un)check this to re-run the checklist action*

### CODE REVIEW

**Review by renanrodrigo — APPROVED** [review]

File: features/retry_auto_attach.feature [hunk 1]
```diff
@@ -117,19 +117,14 @@ Feature: auto-attach retries periodically on failures
 
     Examples: ubuntu release
       | release | machine_type  |
-      | xenial  | aws.generic   |
```
Comment (orndorffgrant) [inline features/retry_auto_attach.feature:120]: auto-attach retries should work on aws, right? What changed?

Comment (orndorffgrant) [reply features/retry_auto_attach.feature:120]: Sorry I should read commit messages before commenting :facepalm:

Comment (lucasmoura) [reply features/retry_auto_attach.feature:120]: Done

**Review by orndorffgrant — APPROVED** [review]
Thank you!

**Pull request merged**